.devcontainer/docker-compose.yml

```diff
@@ -76,5 +76,6 @@ services:
       MINIO_ROOT_USER: user
       MINIO_ROOT_PASSWORD: password
       MINIO_CONSOLE_ADDRESS: ":9001"
-    command: server /data
+    entrypoint: sh
+    command: -c 'mkdir -p /data/{bucket1,bucket2,bucket3} && exec minio server /data'
```

Comment:
please use the `postStartCommand` like we already do to create the `fasttrackml` bucket

Reply:
this offers 1) max compatibility with other dev tooling and 2) matches what we do in `tests/integration/docker-compose.yaml'

Reply:
but it also makes us totally inconsistent

Reply:
also, to address your points:
1) we do not _need_ to keep compatibility with other tooling than what we've described in the dev guide — `postStartCommand` is part of the dev container standard, and anyone using a tool that implements this standard will have their buckets created
2) this is actually really badly implemented and sometimes results in weird things in the integration tests — these buckets should be considered fixtures and should be created and removed by the integration tests

Reply:
our goal should actually be to address 2) and make this better

Reply:
for 1) I just think putting it in docker-compose is compatible *both* with devcontainer and plain docker, whereas `postStartCommand` *only* works for devcontainer -- when something can be either more compatible or less with no impact to the devcontainer, why insist on less?

for 2) do you want to just remove this change here and add it to integration tests setup instead?

Reply:
1) because it's not consistent (one bucket is created one way, and another is created another way) and it's a hack
2) yes please, it would make much more sense

Reply:
It is _not_ proprietary! It's part of an [open spec](https://containers.dev) implemented by a growing number of tools. It's not more or less open than the "standard" docker compose spec!

Reply:
moved to test setup